lib/trino-collect/src/main/java/io/trino/collect/cache/EvictableCache.java

```diff
@@ -86,6 +89,11 @@ private static <K, V> LoadingCache<K, V> buildUnsafeCache(CacheBuilder<? super K
         return cacheBuilder.build(cacheLoader);
     }
 
+    private void expireTokens()
+    {
+        tokens.entrySet().removeIf(e -> e.getValue().isExpired());
```

Comment:
```suggestion
        tokens.values().removeIf(Token::isExpired);
```

(The view collections of a `Map` write-through to the map itself.)